Sources/SwiftInspectorVisitors/ClassVisitor.swift

```diff
@@ -137,15 +130,15 @@ public final class ClassVisitor: SyntaxVisitor {
 
   // MARK: Private
 
-  private let parentTypeName: String?
+  private let parentType: TypeDescription?
```

Comment:
Because in the following sample, `InnerClass` has a parent type of `OuterClass`. This helps us create a fully-qualified name `OuterClass.InnerClass`.
```
final class OuterClass {
  final class InnerClass {}
}
```

Note that the outer type doesn't _need_ to be a class. It could be an enum or struct.

Reply:
Hmm I'm not sure I follow. Maybe to be more clear, what is the difference between `self.parentType` and `self.classInfo?.parentType`? An example may help get us on the same page.

Reply:
Aha! I'm pretty sure they end up being the same. But we don't have a `self.classInfo` at `init`, and I needed somewhere to store the `parentType` until we had one.